set.mm

```diff
@@ -12167,6 +12214,31 @@ logical connectives we have introduced (either axiomatically or by a
     ( wfal wxo wtru wn wb df-xor falbifal xchbinx nottru bitri ) AABZCDAKAAECAA
     FGHIJ $.
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Joint denial
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $( A ` -\/ ` identity.  (Contributed by Remi, 25-Oct-2023.) $)
```

Comment:
Maybe we should be a little bit more verbose here (I know the other theorems of this kind have also these short comments). I would suggest:

``A ` -\/ ` identity: true nor true is false.``

Reply:
In general, I favor precise comments, like the one @avekens wrote, but in this particular case, I think the current comment is sufficient, similar to other comments in the section on truth tables.